Makefile

```diff
@@ -13,7 +13,7 @@ NAME = Talos
 
 ARTIFACTS := _out
 TOOLS ?= ghcr.io/talos-systems/tools:v0.10.0-alpha.0-1-g67314b1
-PKGS ?= v0.10.0-alpha.0-13-gc14eb99
```

Comment:
you also need to update:

* `hack/release.toml`
* `pkg/machinery/constants/constants.go` with new kernel version

Reply:
thanks